### PR TITLE
Improved docs for std.range.transversal

### PR DESCRIPTION
**Pull request by JackStouffer**

### CODE REVIEW

File: std/range/package.d [hunk 1]
```diff
@@ -5955,9 +5955,16 @@ FrontTransversal!(RangeOfRanges, opt) frontTransversal(
 }
 
 /**
-   Given a range of ranges, iterate transversally through the the $(D
-   n)th element of each of the enclosed ranges. All elements of the
-   enclosing range must offer random access.
+    Given a range of ranges, iterate transversally through the the
```
Comment (quickfur) [inline std/range/package.d]: Repeated word "the".

File: std/range/package.d [hunk 2]
```diff
@@ -5955,9 +5955,16 @@ FrontTransversal!(RangeOfRanges, opt) frontTransversal(
...
+    `n`th element of each of the enclosed ranges.
+
+    Params:
+        opt = Controls the assumptions the function makes about them lengths
```
Comment (quickfur) [inline std/range/package.d]: s/them/the/.

**Comment by JackStouffer** [comment]
@quickfur Fixed

**Comment by quickfur** [comment]
LGTM, thanks!

**Comment by quickfur** [comment]
Auto-merge toggled on

**Pull request merged**